Cargo.toml

```diff
@@ -20,14 +20,14 @@ exclude = ["python"]
 members = ["ballista-cli", "ballista/client", "ballista/core", "ballista/executor", "ballista/scheduler", "ballista/tests", "benchmarks", "examples"]
 
 [workspace.dependencies]
-arrow = {version = "43.0.0"}
```

Comment:
We can use patch as well?

https://github.com/coralogix/dataprime-query-engine/blob/master/Cargo.toml#L159